src/dotnet-test-nunit/project.json

```diff
@@ -4,7 +4,7 @@
   "title": "NUnit .NET Core Runner",
   "description": "NUnit console and Visual Studio runner for .NET Core and .NET 4.5.1+",
   "name": "dotnet-test-nunit",
-  "version": "3.3.0",
+  "version": "3.4.0",
```

Comment:
Maybe we can add "dotnet core" in tags array and erase "dnx": https://github.com/nunit/dotnet-test-nunit/blob/31022ae3eb06dedddc4ab973f8ea9e84c9cbc3e0/src/dotnet-test-nunit/project.json#L15 :)